.automation.conf/run-tempest.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Script based on tempest section of Kayobe multinode deployment script
+# plus https://wiki.stackhpc.com/doc/using-kayobe-automation-to-run-tempest-7yAEJw2eHb
+
+set -euxo pipefail
+
+KAYOBE_CONFIG_BASE_PATH=$KAYOBE_CONFIG_PATH/../../
+
+set +x
+export KAYOBE_AUTOMATION_SSH_PRIVATE_KEY=$(cat ~/.ssh/id_rsa)
+export TEMPEST_OPENRC="$(< $KAYOBE_CONFIG_PATH/../kolla/admin-openrc.sh)"
```

Comment:
This would be problematic for environments that don't use the default `id_rsa` key.

Reply:
We can try setting this conditionally. If environment variable is already defined, use the value of it. Otherwise, use these default values.
But don't we usually use ``public-openrc.sh`` as a default?

Reply:
Switched to the public openrc and changed it so it won't override the variables if they're already defined. That's as good as we can get I think without converting it all to a playbook or something